tests/graph/test_edges.py

```diff
@@ -190,6 +190,29 @@ def test_add_edge_from_empty_no_data(self):
         res = graph.add_edges_from_no_data([])
         self.assertEqual([], res)
 
+    def test_extend_from_weighted_edge_list_empty(self):
+        graph = retworkx.PyGraph()
```

Comment:
don't need DAG tests?

Reply:
This PR only added the option to `PyGraph`  so there are no changes on `PyDiGraph`/`PyDAG`. There wasn't as pressing a need for this feature on directed graphs so I figured we could just do it later if people wanted it.

Reply:
I added PyDiGraph/PyDAG support in https://github.com/Qiskit/retworkx/pull/118/commits/0220e88d56573e4aee432622361943f95e605279 it felt odd to omit it (and also makes the interface consistent in #252 for documenting directed and undirected graphs).